### DISCUSSION TITLE
How to deal with the lazyloading problem?

### DISCUSSION THREAD

**Discussion by BytePrince** [adityatelange/hugo-PaperMod#641] [Q&A]
I am used to write some articles with some pictures.But I found that when I use `![]()` to insert some pictures(In order to give me a see-is-what-you-get effect when I write locally),Lazy loading properties will not be automatically generated there.I only use this way of writing `{{< figure src="xxxxxxx.jpeg" alt="xxxx"  attrlink="xxxxxx" class="align-center" >}}`, lazy loading will take effect.
How can I modify the default template so that pictures rendered with `![]()` have lazy loading attributes automatically?
Thanks!

**Comment by danielfdickinson** [comment]
This isn't a final answer, but I have a WIP at <https://github.com/danielfdickinson/hugo-dfd-responsive-images> that might be of interest.

**Comment by adityatelange** [comment]
@BytePrince We alrady have lazy-loading enabled everywhere

https://github.com/adityatelange/hugo-PaperMod/blob/386d84578278d44f4f70c9ed2cebba9623c4007b/layouts/_default/_markup/render-image.html#L1

and in figure shortcode

https://github.com/adityatelange/hugo-PaperMod/blob/386d84578278d44f4f70c9ed2cebba9623c4007b/layouts/shortcodes/figure.html#L8

**Reply by BytePrince** [reply]
I found that when I use `![]()` to insert some pictures(In order to give me a see-is-what-you-get effect when I write locally),Lazy loading properties will not be automatically generated there.
:)

**Reply by danielfdickinson** [reply]
Have you a custom ``render-image`` hook or a really old version of Hugo? Using the stock theme this wont' happen, as pointed out with the code for ``render-image`` in the theme.
